core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,40 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
```

Comment:
well, PathEntry is (for me) in the sense of expression more closer to the LinkEntry. FileEntry more likely tells what the entry represents... But this is only my feeling. 
WDYT @bartoszmajsak